plugin/core/windows.py

```diff
@@ -393,12 +382,14 @@ def stderr_message(self, message: str) -> None:
         """
         pass
 
-    def log(self, message: str, params: Any, log_payload: bool = True) -> None:
+    def log(self, message: str, params: Any) -> None:
 
         def run_on_async_worker_thread() -> None:
             nonlocal message
-            if log_payload:
-                message = "{}: {}".format(message, params)
+            params_str = str(params)
+            if len(params_str) >= settings.log_max_size:
+                params_str = '<params with {} characters>'.format(len(params_str))
```

Comment:
Hmm, this diff strips away all the information. The linked issue tells to "cut off" the params. That is however not possible given that we're highlighting it. It would screw up the syntax parser.

We could disable highlighting the params.